### ISSUE TITLE
Add arm64 dylib version for macOS

### ISSUE THREAD

**Issue by emlai** [ChrisDill/Raylib-cs#127]
### Issue description

This package only provides `libraylib.dylib` for `osx-x64`, but not for arm64. Trying to use this on a M1 Mac causes the following error:

```
System.DllNotFoundException: Unable to load shared library 'raylib' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable:
dlopen(libraylib, 1): image not found
```

After manually adding the arm64 dylib (installed via `brew install raylib`) under `bin/Debug/net6.0/runtimes/osx.11.0-arm64/native/` and making the related config changes, the code runs without errors.

### Environment

- OS: macOS Big Sur 11.6.5

### Code example

I used the example code from the readme.

**Comment by ChrisDill** [comment]
@emlai Raylib-cs uses the native libs included in the official raylib releases so this is not currently supported. Though we are considering custom builds in #118 which could allow other platforms to be added.

**Comment by emlai** [comment]
It seems that starting from raylib 4.2.0, they do provide a universal (x64 + arm64) dylib in their official release:

```
$ file /Users/emlai/Downloads/raylib-4.2.0_macos/lib/libraylib.dylib 
/Users/emlai/Downloads/raylib-4.2.0_macos/lib/libraylib.dylib: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit dynamically linked shared library x86_64] [arm64]
/Users/emlai/Downloads/raylib-4.2.0_macos/lib/libraylib.dylib (for architecture x86_64):	Mach-O 64-bit dynamically linked shared library x86_64
/Users/emlai/Downloads/raylib-4.2.0_macos/lib/libraylib.dylib (for architecture arm64):	Mach-O 64-bit dynamically linked shared library arm64
```

**Comment by ChrisDill** [comment]
@emlai Thanks for looking into this. In that case it should be included in the 4.2 release.

**Comment by leftbones** [comment]
Any progress on this? Still unable to build any project using Raylib-cs or Raylib-CsLo in Visual Studio Mac.

Testing on a MacBook Air M1.
